builder-image/README.md

```diff
@@ -3,8 +3,10 @@ Build with docker:
 
 * JDK11: `cekit --redhat build docker`
```

Comment:
not related to this PR but we can get rid of the JDK11 instructions

Reply:
Removed. Thanks.